tests/conditional_compilation/conftest.py

```diff
@@ -59,6 +63,16 @@ def pytest_addoption(parser):
         type=Path,
         help="Path to OpenVINO repository root directory",
     )
+    parser.addoption(
+        "--omz_repo",
+        type=Path,
+        help="Path to Open Model Zoo repository root directory",
+    )
+    parser.addoption(
+        "--omz_cache_dir",
+        type=Path,
+        help="Path to Open Model Zoo cache directory",
```

Comment:
add default=abs_path('../_omz_out/cache') path

Reply:
@just-sparta Added default value, but we don't use abs_path() in conftest.py, because it's defined in get_testdata.py, and it's not a good idea to import abs_path() in conditional_compilation from stress_tests. Instead, we use native Path.resolve() here.
/cc @ishariko